azure-pipelines.yml

```diff
@@ -43,7 +40,6 @@ steps:
     displayName: "Lint"
 
   - script: |
-      go test -v -race ./core
-      go test -v -race ./cmd
+      go test -v -race ./...
     workingDirectory: "$(modulePath)"
-    displayName: "Test"
+    displayName: "Test"
```

Comment:
Can you add newline? (makes the file grepable).
the setting is called "Insert Final Newline" in vscode GUI or `"files.insertFinalNewline": true` in the config json.